src/Layer.js

```diff
@@ -18,24 +18,23 @@ const renderers = new WeakMap();
 *
 * A Layer is the primary way to visualize geospatial data.
 *
-* To create a layer a {@link carto.source.Base|source} and {@link carto.Viz|viz} are required:
+* To create a layer a {@link carto.source|source} and {@link carto.Viz|viz} are required:
```

Comment:
The current baseclass for all is `carto.source.Base`, which is marked as @abstract, but it has no methods. Maybe it could be renamed to be `carto.source.Source`

Reply:
As David says, carto.source is a namespace. I know that the base class is carto.source.Base, but I think we could mislead people to error if we link to this class directly in this line. What I did here is to link the source namespace, because once you click on it, you go directly to the sources section and you can see the different sources.